spec/unit/procto/to_proc_spec.rb

```diff
@@ -0,0 +1,31 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Procto, '.to_proc' do
+  include_context 'procto'
+
+  subject { proc.call(text) }
+
+  let(:proc) { klass.to_proc }
+
+  let(:klass) do
+    Class.new do
+      include Procto.call
+
+      def initialize(text)
+        @text = text
+      end
+
+      def call
+        "Hello #{@text}"
+      end
+    end
+  end
+
+  it { should eql('Hello world') }
+
+  it 'exposes a lambda proc' do
+    expect(proc).to be_a_lambda
```

Comment:
Well I added this assertion because, otherwise, mutant tells me that it can remove `to_proc` from the singleton method. Normally this would be fine, but since `Procto` is a library and the name says `to_proc` I think it should uphold the type it is saying it will provide.